src/main/resources/api/service_openapi.yaml

```diff
@@ -768,6 +941,13 @@ components:
           schema:
             $ref: '#/components/schemas/RoleBindingList'
 
+    CreatedControlledGoogleBucketResponse:
```

Comment:
I am thinking we want a single structure that we return on resource retrieve and enumeration. Should we use that same structure for these responses?
The benefit is that the client has one object to navigate. The bad part is having to navigate when there is no reason.

Reply:
I think it's straightforward for our union type to include by reference each of the individual response types.